frontend/src/pages/registrationPage.tsx

```
@@ -0,0 +1,92 @@
+import './registrationPage.css'
+import {useState} from "react";
+
+export default function RegistrationPage() {
+
+    const [isChecked, setIsChecked] = useState<boolean[]>([true, false])
+
+    function handleCheckboxChange(checkboxNumber:number) {
+        if (checkboxNumber === 0) {
```

dein initial value ist bereits true, false.
du könntest überprüfen ob checkboxNumber NICHT dem Wert entspricht, und dann auf false, true setzen.
"Fail fast"